server/src/bootstrap/config/base.ts

```diff
@@ -38,6 +42,30 @@ const baseSchema: Schema<BaseConfig> = {
     default: 'https://ask.gov.sg',
     env: 'HOST_URL',
   },
+  searchHost: {
+    doc: 'Host for opensearch service',
+    format: String,
+    default: 'localhost',
+    env: 'OS_HOST',
```

Comment:
Try to keep the naming of your env vars consistent with the field names used, so ie, `search.host` or `searchHost` should correspond to `SEARCH_HOST`